src/java.base/share/classes/java/lang/invoke/DirectMethodHandle.java

```diff
@@ -86,7 +86,7 @@ static DirectMethodHandle make(byte refKind, Class<?> refc, MemberName member, C
         if (!member.isField()) {
             // refKind reflects the original type of lookup via findSpecial or
             // findVirtual etc.
-            switch (refKind) {
+            return switch (refKind) {
                 case REF_invokeSpecial: {
```

Comment:
Is ':' preferred here (and other cases too) instead of "->"?

Reply:
My mistake. I've replaced the colon now with the lambda operator. See a8706b0

Reply:
> My mistake. I've replaced the colon now with the lambda operator.

Drive by comment, in term of name, `->` is the arrow operator not the lambda operator.
- lambda = parameters + arrow + code
- arrow case =  case + arrow + code

The difference is important because a lambda is a function while an arrow case is a block.

Reply:
Ah OK, good to know. Thanks Remi!